src/Simulation/TargetDefinitions/TargetPackages/Common.Package.props

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="..\..\Common\AssemblyCommon.props" />
+  <Import Project="..\..\Common\DebugSymbols.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <QsharpDocsGeneration>true</QsharpDocsGeneration>
```

Comment:
@cgranade Because this common props file is used by all of the targeting packages, I assume this means that we end up re-generating the docs for commonly defined intrinsics (like `X`) more than once. How does this affect the doc generation pipeline? Should I move this so that it is only true for Qsharp.Core and let that be the single source of generated docs?